File: CS/EF/Controllers/EFGridModeController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using EF.Models;

namespace EF.Controllers {
    public class EFGridModeController : Controller {
        NorthwindContext db = new NorthwindContext();

        public ActionResult Index() {
            return View();
        }
        
        public ActionResult GridViewPartial() {
            return PartialView(db.Orders.ToList());
        }
    }
}

File: VB/Controllers/EFGridModeController.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Linq
Imports System.Web.Mvc
Imports EF.Models

Namespace EF.Controllers
	Public Class EFGridModeController
		Inherits Controller
		Private db As New NorthwindContext()

		Public Function Index() As ActionResult
			Return View()
		End Function

		Public Function GridViewPartial() As ActionResult
			Return PartialView(db.Orders.ToList())
		End Function
	End Class
End Namespace